src/services/api/messages.rs

```diff
@@ -43,3 +43,56 @@ impl StarknetMessageToL1 {
         finalized_hash.as_slice().to_vec()
     }
 }
+
+#[test]
+fn create_starknet_message_to_l1() {
+    let from_address: Address = Address(42.into());
+    let to_address: Address = Address(1729.into());
+    let payload: Vec<Felt> = Vec::from([Felt::new(1), Felt::new(2), Felt::new(3), Felt::new(4)]);
```

Comment:
Please remove this type annotations.

Reply:
Done!

Reply:
I had removed the annotations from payload but not from the from_address and to_address. Now it's done done. Please check again!